### PR TITLE
Ianmacleod/add mistral

### PR DESCRIPTION
**Pull request by ian-scale**
adding mistral endpoint support for completions

### CODE REVIEW

File: model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py [hunk 1]
```diff
@@ -117,6 +117,8 @@
         "falcon-7b-instruct": "tiiuae/falcon-7b-instruct",
         "falcon-40b": "tiiuae/falcon-40b",
         "falcon-40b-instruct": "tiiuae/falcon-40b-instruct",
+        "mistral-7b": "mistralai/Mistral-7B-v0.1",
```
Comment (yixu34) [inline model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py:120]: Do we need to add additional validation/tests? Also, think we need to update the EGP APIs?

Also we may want to update our Model Zoo docs to have some more granularity, to reflect the fact that not all models are supported for all frameworks.

Comment (yixu34) [inline model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py:120]: Do we need to add additional validation/tests? Also, think we need to update the EGP APIs?

Also we may want to update our Model Zoo docs to have some more granularity, to reflect the fact that not all models are supported for all frameworks.

**Review by yixu34 — APPROVED** [review]

File: model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py (hunk 1 above)
Comment (ian-scale) [reply model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py:120]: Oh yes, the EGP APIs will also need to change. And I will add the model-zoo comment and add a supported frameworks column to the docs.

Comment (ian-scale) [reply model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py:120]: as far as testing, we can add synthetic monitors for completions for the new models. I will take out a new pr for this.

Comment (yunfeng-scale) [inline model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py:120]: update L494 about context window?

**Pull request merged**